java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/util/BatchJobUtil.scala

```diff
@@ -0,0 +1,46 @@
+package com._4paradigm.openmldb.taskmanager.util
+
+import java.io.File
+
+object BatchJobUtil {
+
+  /**
+   * Get the default batch jor from presupposed directories.
+   *
+   * @return the path of openmldb-batchjob jar
+   */
+  def findLocalBatchJobJar(): String = {
```

Comment:
what's the working directory for those funs ?

Reply:
It would be the directory of shell script if we use the pre-built package. Then we will check for `../jars/` by default.

It would be the directory of maven parent-module  if we run in IDE. Then we will check for `./openmldb-batchjob/target/` by default.